src/burnchains/mod.rs

```diff
@@ -342,13 +347,23 @@ impl PoxConstants {
             pox_participation_threshold_pct,
             sunset_start,
             sunset_end,
+            v1_unlock_height,
             _shadow: PhantomData,
         }
     }
     #[cfg(test)]
     pub fn test_default() -> PoxConstants {
         // 20 reward slots; 10 prepare-phase slots
-        PoxConstants::new(10, 5, 3, 25, 5, 5000, 10000)
+        PoxConstants::new(10, 5, 3, 25, 5, 5000, 10000, u32::max_value())
+    }
+
+    /// Returns the PoX contract that is "active" at the given burn block height
+    pub fn active_pox_contract(&self, burn_height: u64) -> &'static str {
+        if burn_height >= (self.v1_unlock_height as u64) {
+            "pox-2"
```

Comment:
Do we want to label the PoX contract by the Stacks major/minor version, or by the contract version?  i.e. `pox-2` vs `pox-2.1`?  Either one is fine with me, just as long as it's consistent.

Reply:
My preference is by the contract version -- it's not the case that future versions of the Stacks chain (if they exist) will necessarily create new PoX contracts, so it'd be kind of confusing that `Stacks 2.9` uses `pox-2.4` (or whatever the versioning mismatch becomes).

Reply:
However, I could be convinced otherwise if people believe one or the other will create more confusion.

Reply:
Up to you. I'm fine with it as-is as long as we believe users won't get confused.